src/vdaf/mastic.rs

```diff
@@ -5,19 +5,22 @@
 //! [draft-mouris-cfrg-mastic-01]: https://www.ietf.org/archive/id/draft-mouris-cfrg-mastic-01.html
 
 use crate::{
+    bt::{BinaryTree, Path},
```

Comment:
Can we remove `#![allow(dead_code)]` etc. from the `bt` module now that this is using it?

Reply:
No, we're not using the main BT API so it's still dead.

Reply:
> Can we remove `#![allow(dead_code)]` etc. from the `bt` module now that this is using it?

We ended up not needing some of what @armfazh prepared. We should prune the code we're not using, but we should wait until until Mastic is complete, just in case we end up wanting it.

> Relatedly, I think we ought to make the `bt` module public now, since `BinaryTree` appears in the arguments of the public `Vidpf::eval_with_cache()` method.

I'd suggest we make `eval_with_cache()` `pub(crate)` for now.